src/main/java/nl/ordina/jobcrawler/controller/JobcrawlerController.java

```diff
@@ -1,73 +1,95 @@
 package nl.ordina.jobcrawler.controller;
 
+
 import nl.ordina.jobcrawler.SearchRequest;
 import nl.ordina.jobcrawler.SearchResult;
-import nl.ordina.jobcrawler.model.Vacancy;
 import nl.ordina.jobcrawler.controller.exception.VacancyNotFoundException;
+import nl.ordina.jobcrawler.model.Skill;
+import nl.ordina.jobcrawler.model.Vacancy;
+import nl.ordina.jobcrawler.service.SkillService;
 import nl.ordina.jobcrawler.service.VacancyService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
-import java.util.*;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
 
 
 @CrossOrigin(origins = "http://localhost:4200")
 @RestController
 public class JobcrawlerController {
 
     private VacancyService vacancyService;
+    private SkillService skillService;
 
     @Autowired
-    public JobcrawlerController(VacancyService vacancyService) {
+    public JobcrawlerController(VacancyService vacancyService, SkillService skillService) {
         this.vacancyService = vacancyService;
+        this.skillService = skillService;
     }
 
-    // Post mappings
-
     @PostMapping("/searchrequest")
     public SearchResult searchRequest(@RequestBody SearchRequest request) {
-        return new SearchResult(request, vacancyService.getAllJobs());
+        return new SearchResult(request, vacancyService.getAllVacancies());
```

Comment:
There are no comments to be found for any of the endpoints.
A comment should state: What is the endpoints purpose? Are there side effects? What is the outcome?
Is this retrieving the  vacancy table contents from the database? Pleas add this to the comment.

Reply:
One thing to be discussed if we really want to put all the comments in the controller. As we want to have functionality and files separated, describing everything in the controller would lead to changing a file and changing its comments in another file. 

As I understand it the controller should only have the REST services, so maybe only have the REST services comments in this file? To be discussed in the meeting.

Reply:
more comments per end point

Reply:
done